posdealers/03-sales/04c-masterdata-creation.md

```diff
@@ -1,130 +1,238 @@
 ---
 slug: /posdealers/get-started/sales/masterdata-creation
-title: Bereitstellung von Stammdaten
+title: Stammdaten eines KassenBetreibers
 ---
 
-# Bereitstellung von Stammdaten eines KassenBetreibers
+# Stammdaten eines KassenBetreibers
 
-Dieses Dokument beschreibt den Minimalumfang jener Stammdaten eines KassenBetreibers, welcher eingepflegt und gewartet werden muß, um Produkte und Features von fiskaltrust nutzen zu können. Daten, welche in diesem Dokument nicht aufgelistet sind, werden von fiskaltrust entweder im Hintergrund automatisiert erzeugt (z.B. TSE Daten für den DSFinV-K Export) oder sind gesetzlich nicht gefordert, oder sind optional und somit für die ordnungsgemäße Funktion nicht zwingend erforderlich. Des weiteren wird beschrieben, wie diese Daten in das fisaltrust.Portal eingepflegt werden können, und wer bzw. welche Rolle (KassenHersteller, KassenHändler, KassenBetreiber, Steuerberater) für die Erzeugung und Wartung dieser Daten verantwortlich ist.
+Dieses Dokument beschreibt Anforderungen an die Stammdaten eines KassenBetreibers wie Daten des Unternehmens (im weiteren Firma genannt) oder Benutzerdaten. Diese Daten müssen im fiskaltrust.Portal mindestens eingegeben, kontrolliert und gepflegt werden. Nur so sind Produkte und Features von fiskaltrust zu nutzen. Weiter gelingen nur mit korrekt eingegebenen Daten auch Exporte.<br>
```

Comment:
Be careful with HTML tags: Docusaurus will fail building this page because of the `<br>`. All tags must be escaped, so only `<br />` works here.

```suggestion
Dieses Dokument beschreibt Anforderungen an die Stammdaten eines KassenBetreibers wie Daten des Unternehmens (im weiteren Firma genannt) oder Benutzerdaten. Diese Daten müssen im fiskaltrust.Portal mindestens eingegeben, kontrolliert und gepflegt werden. Nur so sind Produkte und Features von fiskaltrust zu nutzen. Weiter gelingen nur mit korrekt eingegebenen Daten auch Exporte.<br />
```